### PR TITLE
Make citest-jasmine pass locally

### PR DESCRIPTION
**Pull request by etpinard**
resolves https://github.com/plotly/plotly.js/issues/666 (hopefully)

How good does this

image

feel?

cc @rreusser @n-riesco @alexcjohnson @monfera

### CODE REVIEW

File: test/jasmine/tests/download_test.js [hunk 1]
```diff
@@ -45,11 +40,11 @@ describe('Plotly.downloadImage', function() {
 
     it('should create link, remove link, accept options', function(done) {
         downloadTest(gd, 'jpeg', done);
-    });
+    }, LONG_TIMEOUT_INTERVAL);
```
Comment (etpinard) [inline test/jasmine/tests/download_test.js]: See https://github.com/jasmine/jasmine/commit/68ba5b6d48624c063b4eeae1bdd63c5aa84a3177 for example.

File: test/jasmine/tests/hover_label_test.js [hunk 2]
```diff
@@ -749,7 +749,7 @@ describe('hover on fill', function() {
         }).then(function() {
             return assertLabelsCorrect([237, 218], [266.75, 265], 'trace 1');
         }).then(function() {
-            return assertLabelsCorrect([237, 251], [247.7, 254], 'trace 0');
+            return assertLabelsCorrect([237, 240], [247.7, 254], 'trace 0');
```
Comment (etpinard) [inline test/jasmine/tests/hover_label_test.js]: cc @alexcjohnson these cases :arrow_up: were tuned for FF on CI and Chrome locally. Now, they work locally in FF and Chrome as well as FF on CI.

**Comment by etpinard** [comment]
Note that `npm run test-jasmine` (i.e. running all the test in Chrome locally) still fails from time to time. Something is up with in gl2d tests. This fix will be for a different PR.

**Comment by rreusser** [comment]
This all works except for one test:

```
Firefox 50.0.0 (Mac OS X 10.11.0): Executed 716 of 1539 SUCCESS (0 secs / 1 min 30.653 secs)
Firefox 50.0.0 (Mac OS X 10.11.0) update menus interactions should relayout FAILED
	Expected 11 to be less than 11.
	assertItemDims@/var/folders/5s/mm8_jrls0zd34zkb1z71xb940000gp/T/tests/updatemenus_test.js:701:0 <- /var/folders/5s/mm8_jrls0zd34zkb1z71xb940000gp/T/40ed0cfc29821315921e9b4f89111e80.browserify:188035:9
	require<["/Users/rreusser/plotly/plotly.js/test/jasmine/tests/updatemenus_test.js"]</</</<@/var/folders/5s/mm8_jrls0zd34zkb1z71xb940000gp/T/tests/updatemenus_test.js:569:0 <- /var/folders/5s/mm8_jrls0zd34zkb1z71xb940000gp/T/40ed0cfc29821315921e9b4f89111e80.browserify:187903:13
Firefox 50.0.0 (Mac OS X 10.11.0): Executed 1539 of 1539 (1 FAILED) (4 mins 3.606 secs / 4 mins 0.075 secs)
```

That's my fault. Not clear why it's failing, but of course 569 is the *one* line out of many that I'm responsible for: [`updatemenus_test.js:569`](https://github.com/plotly/plotly.js/blame/fix-citest-jasmine/test/jasmine/tests/updatemenus_test.js#L569)

**Comment by rreusser** [comment]
For reference: conclusion after talking with @etpinard: change the `11` in the width comparison function to a `12`. It's effectively hard-coding font layout on different platforms, which makes it an unsurprising failure. 11 -> 12 simply increases the tolerance just a touch. Otherwise tests pass for me 💃

**Pull request merged**